docker/composer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer/composer
```

Comment:
My point is, that the composer-container should have the same extensions as the php (harvester)-container, so that if you try to install a package that requires intl, and intl does not exist in the composer-container (and thus also does not exist in the php-container) it will fail.

This means that you cannot have a system-wide composer-container to use every time, without having to setup extensions every time.

I think we need a company-wide talk about those types of containers. Whether or not we want them, and if we do, how they should be implemented. Another take on it is a different docker-compose-file with them (eg. docker-compose.dev.yml), but that requires aliasing to be convenient.